dapps/marketplace/src/pages/user/_EditModal.js

```diff
@@ -52,13 +51,15 @@ class EditProfileModal extends Component {
           <div className="row">
             <div className="col-6">
               <ImageCropper
-                onChange={async avatar => {
+                onChange={async avatarBase64 => {
                   const { ipfsRPC } = this.props.config
-                  const uploadedImages = await uploadImages(ipfsRPC, [avatar])
+                  const uploadedImages = await uploadImages(ipfsRPC, [
+                    avatarBase64
```

Comment:
Do you want to add a comment explaining why we upload base64 encoded image ?
This is just because of legacy.

Do you actually think it would be worthwhile uploading as binary moving forward ?
On the plus side, it would reduce a bit size and computation needed on client to decode. But may not be worth the extra complexity in the code to handle legacy base64 format ?